docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Server Configuration And Management/Configuration Options/Phone Home/PhoneHome.adoc

```diff
@@ -65,9 +45,59 @@ would only gather data specific to Payara and has no contact with any
 running applications, or indeed anything on your machine other than the
 mentioned attributes.
 
+== Disabling and Enabling PhoneHome
+
+The phone home service is enabled by default when starting a Payara
+Server or Payara Micro. The phone home
+service can easily disabled in four different ways.
+
+[[enable-phone-home]]
+== Enable the Phone Home service
+
+====
+asadmin> enable-phone-home
+====
+
+[[disable-phone-home]]
+== Disable the Phone Home service
+
+[[disable-with-asadmin]]
+=== Using Asadmin
+
+Run this asadmin command then restart the server to disable PhoneHome
+
+====
+asadmin> disable-phone-home
+====
+
+[[disable-with-domainxml]]
+=== Disabling in the _domain.xml_ File
+
+- Open `domain.xml` with a text editor
+- Find the tag `<config name="server-config">
+- Within that tag, add the following tag:
+- <phone-home-runtime-configuration enabled="false"/>
```

Comment:
We discourage from manually editing the domain.xml file, so I'd remove this section.
```suggestion
```

Reply:
I was tempted to exclude this myself when I saw it